inst/traduire/en-translation.json

```diff
@@ -87,5 +87,7 @@
     "QUARTER_4": "Quarter 4",
     "VALIDATION_MULTIPLE_LEVELS_DETAIL": "year {{year}} has area levels {{levels}}",
     "VALIDATION_MULTIPLE_LEVELS": "Data can only be for regions at a single area level per year. In uploaded data {{detail}}.",
-    "FAILED_ZIP_REHYDRATE_SUBMIT": "Cannot load from this zip file, archive missing required information. Please regenerate output zip and try again."
+    "FAILED_ZIP_REHYDRATE_SUBMIT": "Cannot load from this zip file, archive missing required information. Please regenerate output zip and try again.",
+    "TABLE_SEX_BY_AREA": "Sex by area",
```

Comment:
Need fr and pt for these too? We'll be fetching output metadata when language selection changes, and updating view from that as we already do for other output metadata I assume?

Reply:
Thanks! Yes I do